bazel/emscripten_toolchain/env.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-export ROOT_DIR=`(pwd -P)`
+export ROOT_DIR=${EXT_BUILD_ROOT:-`pwd -P`}
```

Comment:
Maybe switch to `$(pwd -P)`  synctax to match the changes above?

Reply:
Done.